app/models/facility.rb

```diff
@@ -106,7 +106,7 @@ class Facility < ApplicationRecord
   delegate :protocol, to: :facility_group, allow_nil: true
   delegate :organization, :organization_id, to: :facility_group, allow_nil: true
   delegate :follow_ups_by_period, to: :patients, prefix: :patient
-  delegate :district_region?, :block_region?, :facility_region?, :region_type, to: :region
+  delegate :district_region, :district_region?, :block_region?, :facility_region?, :region_type, to: :region
```

Comment:
Why add a delegation for `district_region`? Is it used in this PR? I see we're still forming the label using facility group.